Sources/SmokeOperationsHTTP1Server/SmokeHTTP1Server+runAsOperationServer.swift

```diff
@@ -299,4 +299,124 @@ public extension SmokeHTTP1Server {
+                                              invocationStrategy: initalizer.invocationStrategy,
+                                              defaultLogger: initalizer.defaultLogger,
+                                              eventLoopProvider: eventLoopProvider,
+                                              shutdownOnSignals: initalizer.shutdownOnSignals)
+        do {
+            try server.start()
+            
+            try await server.untilShutdown()
+            
+            try await initalizer.onShutdown()
+            
+            try await eventLoopGroup.shutdownGracefully()
+        } catch {
+            logger.error("Operations Server lifecycle error: '\(error)'")
+        }
+    }
+    
+    static func runAsOperationServer<InitializerType: SmokeAsyncServerPerInvocationContextInitializer, TraceContextType>(
+        _ factory: @escaping (EventLoopGroup) async throws -> InitializerType) async
+    where InitializerType.SelectorType.DefaultOperationDelegateType.InvocationReportingType == SmokeServerInvocationReporting<TraceContextType>,
+          InitializerType.SelectorType.DefaultOperationDelegateType.RequestHeadType == SmokeHTTP1RequestHead,
+          InitializerType.SelectorType.DefaultOperationDelegateType.ResponseHandlerType ==
+            StandardHTTP1ResponseHandler<SmokeInvocationContext<InitializerType.SelectorType.DefaultOperationDelegateType.InvocationReportingType>> {
+        let eventLoopGroup =
+            MultiThreadedEventLoopGroup(numberOfThreads: System.coreCount)
+        
+        let initalizer: InitializerType
+        do {
+            initalizer = try await factory(eventLoopGroup)
+        } catch {
+            let logger = Logger.init(label: "application.initialization")
+            
+            logger.error("Unable to initialize application from factory due to error - \(error).")
+            
+            return
+        }
+        
+        // initialize the logger after instatiating the initializer
+        let logger = Logger.init(label: "application.initialization")
+        
+        let eventLoopProvider: SmokeHTTP1Server.EventLoopProvider
+        // if the initializer is indicating to create new threads for the server
+        // just use the created eventLoopGroup
+        if case .spawnNewThreads = initalizer.eventLoopProvider {
+            eventLoopProvider = .use(eventLoopGroup)
+        } else {
+            // use what the initializer says
+            eventLoopProvider = initalizer.eventLoopProvider
+        }
+        
+        var handlerSelector = initalizer.handlerSelectorProvider()
+        initalizer.operationsInitializer(&handlerSelector)
+        
+        let handler = OperationServerHTTP1RequestHandler<InitializerType.SelectorType, TraceContextType>(
+            handlerSelector: handlerSelector,
+            contextProvider: initalizer.getInvocationContext, serverName: initalizer.serverName,
```

Comment:
nit: `serverName` should be on the next line.

Reply:
Fixed.